.homeycompose/capabilities/operation_mode_state.json

```diff
@@ -14,49 +17,70 @@
       "id": "idle",
       "title": {
         "en": "Idle",
-        "fr": "Inactif"
+        "fr": "Inactif",
+        "nl": "Inactief",
+        "sv": "Inaktiv",
+        "es": "Inactivo"
       }
     },
     {
       "id": "dhw",
       "title": {
-        "en": "Domestic house water",
-        "fr": "Eau chaude sanitaire"
+        "en": "Domestic hot water",
+        "fr": "Eau chaude sanitaire",
+        "nl": "Sanitair warm water",
+        "sv": "Hushållsvatten",
```

Comment:
"Varmvattenladdning" or "Varmvattendrift".
I vote for the first word as it's used more commonly used by professionals.